src/Loop/Driver.php

```diff
@@ -189,19 +194,29 @@ public function unreference($watcherId);
      *
      * @return void
      */
-    public function storeState($key, $value);
+    public final function storeState($key, $value)
```

Comment:
I think you will hate me, but this is also not PSR2 compliant (should be `final public`), i'm sorry, but if one day if see you in person i will offer you a beer to compensate :)